requirements/requirements.txt

```diff
@@ -78,7 +78,7 @@ django-uuidfield==0.5.0
 # required by ctable
 SQLAlchemy==1.0.9
 alembic==0.6.4
-pyzxcvbn==0.5.0
+git+git://github.com/dimagi/pyzxcvbn.git
```

Comment:
Recommendation: add `#egg=pyzxcvbn` to the end of the URL. It helps pip to know what package will be installed and therefore pip does not need to download it on every run.